examples/hello_world/lib/main.dart

```diff
@@ -2,6 +2,70 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart';
 
-void main() => runApp(const Center(child: const Text('Hello, world!', textDirection: TextDirection.ltr)));
```

Comment:
Revert this file when you are done with it

Reply:
yup, I'm going to remove everything except for the expansion panel and its test from the commit